src/test/java/com/endava/internship/mocking/service/PaymentServiceTest.java

```diff
@@ -1,31 +1,107 @@
 package com.endava.internship.mocking.service;
 
+import com.endava.internship.mocking.model.Payment;
+import com.endava.internship.mocking.model.Status;
+import com.endava.internship.mocking.model.User;
+import com.endava.internship.mocking.repository.PaymentRepository;
+import com.endava.internship.mocking.repository.UserRepository;
+import org.junit.jupiter.api.Assertions;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.ArgumentCaptor;
+import org.mockito.Captor;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+import java.util.Optional;
+import java.util.UUID;
+import java.util.List;
+import java.util.Arrays;
+import java.util.ArrayList;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.verify;
+
 
 @ExtendWith(MockitoExtension.class)
 class PaymentServiceTest {
 
 
+    @Mock
+    PaymentRepository paymentRepository;
+
+    @Mock
+    UserRepository userRepository;
+
+    @Mock
+    ValidationService validationService;
+
+    @InjectMocks
+    PaymentService paymentService;
+
+    @Captor
+    ArgumentCaptor<Payment> captor;
+
+    User user;
```

Comment:
why are you declaring your user her if you are using it in only one method?
It is not better to declare it in given section of your method?

Reply:
Done